__tests__/app.test.js

```diff
@@ -607,3 +607,27 @@ describe("Challenge 9 - DELETE /api/comments/:comment_id", () =>{
     
 
 })
+
+
+
+
+describe("Challenge 10 - GET /api/users", () => {
+    test("responds with 200 and an array of objects with the correct properties", () => {
+
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) => {
+
+            body.users.forEach(user => {
+                expect(user).toHaveProperty('username', expect.any(String))
+                expect(user).toHaveProperty('name', expect.any(String))
+                expect(user).toHaveProperty('avatar_url', expect.any(String))
+            })
+
+        })
+
+    })
+
```

Comment:
If this is an empty array then the tests within the forEach will automatically pass. I would add a length check :)